inspirehep/dojson/common/base.py

```diff
@@ -340,8 +341,55 @@ def urls(self, key, value):
         else:
             description = None
         if isinstance(val.get('u'), str):
-            urls.append({"value": val.get('u'), "description": description})
+            url_dict['value'] = val.get('u')
+            if description:
+                url_dict['description'] = description
+            urls.append(url_dict)
         elif isinstance(val.get('u'), tuple):
-            for u in val.get("u"):
-                urls.append({"value": u, "description": description})
-    return urls
+            for u in val.get('u'):
+                url_dict = {}
+                url_dict['value'] = u
+                if description:
+                    url_dict['description'] = description
+                urls.append(url_dict)
+
+    return inspire_dojson_utils.remove_duplicates_from_list_of_dicts(urls)
+
+
+@hepnames.over('field_categories', '^65017')
+@conferences.over('field_categories', '^65017')
+@institutions.over('field_categories', '^65017')
+@jobs.over('field_categories', '^65017')
+@experiments.over('field_categories', '^65017')
+@hep.over('field_categories', '^650[1_][_7]')
+def field_categories(self, key, value):
+    """Field categories."""
+    value = utils.force_list(value)
+    field_categories = self.get('field_categories', [])
+    for element in value:
+        _scheme = element.get('2')
+        if isinstance(_scheme, (list, tuple)):
+            _scheme = _scheme[0]
```

Comment:
Here we could add `FIXME: Log MultipleSchemesError` or something similar :smile: